src/app/lib/utilities/preprocessor/rules/blockPositioning.js

```diff
@@ -0,0 +1,49 @@
+import deepGet from '../../../../helpers/json/deepGet';
+import deepClone from '../../../../helpers/json/deepClone';
+
+const insertBlockPositioning = (
+  { model },
+  positionArr = [],
+) => {
+  if (!model || typeof model !== 'object') {
+    return null;
+  }
+
+  const amendedJson = model;
+  let incrementPositionAtLevel = 0;
+  Object.keys(model).forEach(key => {
+    if (key !== 'blocks') {
+      return null;
+    }
+
+    return model[key].map((childObj, index) => {
```

Comment:
```suggestion
    model['blocks'].map((childObj, index) => {
```

Got a feeling this should be a `.forEach()` rather than a `.map()`. `map()`is used to create new arrays whereas this logic is editing the `amendedJson` array so doesn't need to generate an array per child on `model.blocks`